test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
What is ambiguous? I don't see the advantage of including totalPrice if we already are specifying price. I think calling it outputRate is just as confusing as calling it quality.